vcd/resource_vcd_security_tag_test.go

```diff
@@ -0,0 +1,170 @@
+package vcd
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"strings"
+	"testing"
+)
+
+func TestAccVcdSecurityTag(t *testing.T) {
```

Comment:
This test crashes on versions that don't support Security tags (<10.3). (needs a conditional skip)
```
=== RUN   TestAccVcdSecurityTag
    resource_vcd_security_tag_test.go:38: Step 1/2 error: Error running apply: exit status 1

        Error: error when setting up security tags - error getting minimum required API version: endpoint '1.0.0/securityTags' requires API version to support at least '36.0'. Maximum supported version in this instance: '35.0'

          with vcd_security_tag.testaccvcdsecuritytag-tag1,
          on terraform_plugin_test.tf line 21, in resource "vcd_security_tag" "testaccvcdsecuritytag-tag1":
          21: resource "vcd_security_tag" "testaccvcdsecuritytag-tag1" {

    testing_new.go:70: Error running post-test destroy, there may be dangling resources: error retrieving Org datacloud security tags
--- FAIL: TestAccVcdSecurityTag (187.40s)
FAIL
```